docs/HelmChart.md

```diff
@@ -0,0 +1,317 @@
+# Deploying GUAC with Helm Chart
+
+This tutorial introduces the full GUAC components deployment and how to deploy
+it with [Kusari's GUAC Helm Chart](https://github.com/kusaridev/helm-charts/tree/main/charts/guac).
```

Comment:
can we add a note here, that since this is not part of the repo there may be drift between documentation, and if anything is not working, it is best to refer to the original repository docs.

Reply:
note added